.clusterfuzzlite/project.yaml

```diff
@@ -0,0 +1,20 @@
+# -----------------------------------------------------------------------------
+# SPDX-FileCopyrightText: Copyright © 2024 bomctl a Series of LF Projects, LLC
+# SPDX-FileName: .clusterfuzzlite/project.yaml
+# SPDX-FileType: SOURCE
+# SPDX-License-Identifier: Apache-2.0
+# -----------------------------------------------------------------------------
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# -----------------------------------------------------------------------------
+---
+language: go
```

Comment:
Is this the sum total of all configuration for clusterfuzzlite? 😆 Is it possible to omit this file and it would be smart enough to figure out it's a go project?